composer.json

```
@@ -22,35 +22,39 @@
     }
   ],
   "require": {
-    "php": "^8.0",
+    "php-64bit": "^8.2",
     "symfony/polyfill-mbstring": "^1.0",
-    "psr/http-message": "^1.0",
-    "myclabs/php-enum": "^1.5"
+    "psr/http-message": "^1.0"
```

```suggestion
    "psr/http-message": "^1.0",
    "ext-zlib": "*"
```

`DeflateContext` comes from zlib

https://github.com/maennchen/ZipStream-PHP/blob/fed05c209ec1c10f0b008db68d457a87dd52ce03/src/File.php#L37-L37

---

good catch!